share/completions/boon.fish

```diff
@@ -0,0 +1,28 @@
+set command boon
+
+complete -c $command -f
+
+complete -c $command -s h -l help -d 'Show [h]elp'
```

Comment:
We don't really do the `[h]` mnemonic, please remove it.

Reply:
Should all other mnemonics be removed as well?

Reply:
I've remove all mnemonics in square brackets.